hack/app_sre_create_image_catalog.sh

```diff
@@ -19,67 +18,33 @@ git clone \
     https://app:${APP_SRE_BOT_PUSH_TOKEN}@gitlab.cee.redhat.com/service/saas-hive-operator-bundle.git \
     $SAAS_OPERATOR_DIR
 
-# remove any versions more recent than deployed hash
-REMOVED_VERSIONS=""
-if [[ "$REMOVE_UNDEPLOYED" == true ]]; then
-    DEPLOYED_HASH=$(
-        curl -s 'https://gitlab.cee.redhat.com/service/saas-hive/raw/master/hive-services/hive.yaml' | \
-            docker run --rm -i evns/yq -r '.services[]|select(.name="hive").hash'
-    )
-
-    delete=false
-    for version in `ls $BUNDLE_DIR | sort -t . -k 3 -g`; do
-        # skip if not directory
-        [ -d "$BUNDLE_DIR/$version" ] || continue
-
-        if [[ "$delete" == false ]]; then
-            short_hash=$(echo $version | cut -d- -f2)
-            short_hash=${short_hash##sha}
-
-            if [[ "$DEPLOYED_HASH" == "${short_hash}"* ]]; then
-                delete=true
-            fi
-        else
-            rm -rf "$BUNDLE_DIR/$version"
-            REMOVED_VERSIONS="$version $REMOVED_VERSIONS"
-        fi
-    done
-fi
 
 # generate bundle
-PREV_VERSION=$(ls $BUNDLE_DIR | sort -t . -k 3 -g | tail -n 1)
+PREV_VERSION=$((cd $SAAS_OPERATOR_BUNDLE_DIR && find -type d | xargs -n 1 basename) | sort -V  | tail -n 1)
```

Comment:
get only directories, -V version sort (0.1.1800, 1.0.0, 1.0.1 ...)

Reply:
I did not know `sort` supported this `-V` option, nice!